andela/ah-aquaman-frontend
# #164069179 reset password link

## What does this PR do?

This PR implements the feature of a user being provided with a reset password link

## Description of the task to be completed

- Enable users to provide their email to reset their password
- Provide the users with a reset password link
- Enable users to change their password

## How should this be manually tested?

- Clone this repository
- cd into the directory Ah-aquaman-frontend
- Check out the branch `ft-reset-password-164069179`
- Run npm install to install all dependencies
- Run npm start to get the server running
- You will view the reset password page at `localhost/reset_password`
- Once you click on the link sent to your email, you are redirected to the page where you actually change your password `localhost/reset_password/token`

## What are the relevant pivotal tracker stories?

[#164069179](https://www.pivotaltracker.com/story/show/164069179)

## Screenshots

Reset password page

![image](https://user-images.githubusercontent.com/42435346/56277650-6b0d1780-610d-11e9-8595-5940274c8f78.png)


Change password page

![image](https://user-images.githubusercontent.com/42435346/55956837-90ada300-5c6d-11e9-843a-9d46425beac9.png)
